nusantara/nusa_datasets/casa/casa.py

```diff
@@ -0,0 +1,155 @@
+                    "sentence": datasets.Value("string"),
+                    "fuel": datasets.Value("string"),
+                    "machine": datasets.Value("string"),
+                    "others": datasets.Value("string"),
+                    "part": datasets.Value("string"),
+                    "price": datasets.Value("string"),
+                    "service": datasets.Value("string"),
+                }
+            )
+
+        elif self.config.schema == "nusantara_text_multi":
+            features = schemas.text_multi_features(["positive", "neutral", "negative"])
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        train_csv_path = Path(dl_manager.download_and_extract(_URLS["train"]))
+        validation_csv_path = Path(dl_manager.download_and_extract(_URLS["validation"]))
+        test_csv_path = Path(dl_manager.download_and_extract(_URLS["test"]))
+
+        data_dir = {
+            "train": train_csv_path,
+            "validation": validation_csv_path,
+            "test": test_csv_path,
+        }
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": data_dir["train"],
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "filepath": data_dir["test"],
+                    "split": "test",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "filepath": data_dir["validation"],
+                    "split": "dev",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+        df = pd.read_csv(filepath, sep=",", header="infer").reset_index()
+        if self.config.schema == "source":
+            for row in df.itertuples():
+                entry = {"index": row.index, "sentence": row.sentence, "fuel": row.fuel, "machine": row.machine, "others": row.others, "part": row.part, "price": row.price, "service": row.service}
+                yield row.index, entry
+
+        elif self.config.schema == "nusantara_text_multi":
+            for row in df.itertuples():
+                entry = {
+                    "id": str(row.index),
+                    "text": row.sentence,
+                    "labels": [label for label in row[3:]],
+                }
+                yield row.index, entry
+
+
+if __name__ == "__main__":
```

Comment:
Hi @FaridLazuarda, thank you for contributing! Just a little thing, can you remove these two lines? Since this won't be needed anymore after merged

Reply:
Done, appreciate the review!